Components/Core/Spells/Equivalencies.lua

```diff
@@ -376,7 +376,7 @@ TMW.BE = {
 			 155835, -- Bristling Fur
 			 184364, -- Enraged Regeneration
 			 186265, -- Aspect of the Turtle
-			 197268, -- Ray of Hope
+			-197268, -- Ray of Hope
```

Comment:
Made ID agnostic because there are 2 version of the (de?)buff associated with Ray of Hope, one for a positive absorbing ray and a negative absorbing one.